air/tests/test_module/instructions/canon.rs

```diff
@@ -602,3 +617,127 @@ fn canon_map_scalar_with_par() {
 
     assert_eq!(actual_trace, expected_trace);
 }
+
+#[test]
```

Comment:
I would check the contents of the resulting canon streams but given the time constraints this can be done later.

Reply:
I'm agree with you.

Reply:
This test has no resulting canon stream, but we can check intermediate datas.